teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] the constructor should extend the parent's constructor by adding @classroom and a parameter for it. have a look at [this example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance)

Reply:
the specialization you mean. Thanks a lot Changed it.